staging/src/kubevirt.io/api/core/v1/sanitizers.go

```diff
@@ -33,3 +33,13 @@ func SanitizeIP(address string) (string, error) {
 
 	return sanitizedAddress.String(), nil
 }
+
+func SanitizeCIDR(cidr string) (string, error) {
+	ip, net, err := netutils.ParseCIDRSloppy(cidr)
+	if err != nil {
+		return "", fmt.Errorf("not a valid CIDR")
```

Comment:
What about the original error? What does it contain that requires you to add your own text?

Reply:
You are right, there is no need to use a custom error.